Source/EasyNetQ/EasyNetQ.csproj

```diff
@@ -10,10 +10,9 @@
       <PrivateAssets>all</PrivateAssets>
       <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
     </PackageReference>
-    <PackageReference Include="LightInject.Source" Version="6.6.4">
-      <PrivateAssets>all</PrivateAssets>
-    </PackageReference>
     <PackageReference Include="Microsoft.Extensions.Logging.Abstractions" Version="6.0.0" />
+    <PackageReference Include="Microsoft.Extensions.DependencyInjection" Version="6.0.0" />
```

Comment:
I am not that sure about a dependency to `Microsoft.Extensions.DependencyInjection`...

We could consider continuing usage of LightInject.Source and embed https://github.com/seesharper/LightInject.Microsoft.DependencyInjection.

Reply:
@zidad @micdenny What are your thoughts on it?

Reply:
I don't think we should reference any concrete implementation here at all, so Microsoft.Extensions.DependencyInjection should be removed, but it would mean we have to get rid of the possibility to initialize the bus in the EasyNetQ project completely (meaning drop RabbitHutch completely), or move it to another backwards compatible project. 

Since we are making a lot of breaking changes anyway, I would suggest to do the right thing and drop it completely, and not create a backwards compatible project. Or is that too radical @Pliner?

Reply:
> I don't think we should reference any concrete implementation here at all, so Microsoft.Extensions.DependencyInjection should be removed

👍🏻 

> but it would mean we have to get rid of the possibility to initialize the bus in the EasyNetQ project completely (meaning drop RabbitHutch completely)

Not at all. We currently use source based version of LightInject and with help of https://github.com/seesharper/LightInject.Microsoft.DependencyInjection source code we can try implement Microsoft.Extensions.DependencyInjection.Abstraction.

If you are not totally agains this approach I can prepare a branch with these changed on top of this PR.

Reply:
@zidad Ok, you have already removed it. Will attempt to do ^ after a merge of this PR.